File: iniciante/python/1044-multiplos.py
def multiplos():
    valores = list(map(int, input().split(' ')))
    a = valores[0]
    b = valores[1]
    if b % a == 0 or a % b == 0:
        return print('Sao Multiplos')
    else:
        return print('Nao sao Multiplos')


multiplos()

File: iniciante/c/1044-multiplos.c
#include <stdio.h>

int main(){
    int a, b;

    // Entrada do usuário
    scanf("%d", &a);
    scanf("%d", &b);

    // Verifica se a e b são múltiplos entre si
    if (b % a == 0 || a % b == 0){
        printf("Sao Multiplos\n");
    }
    else{
        printf("Nao sao Multiplos\n");
    }

    return 0;
}
